app/models/user.rb

```diff
@@ -0,0 +1,6 @@
+class User < ActiveRecord::Base
+  devise :database_authenticatable, :registerable,
```

Comment:
looks like you are using Devise, we haven't talked about this yet, but can be a little difficult to configure